### PR TITLE
[FrameworkBundle][Validator] Remove remaining deprecations

### PR DESCRIPTION
**Pull request by alexandre-daubois**
| Q             | A
| ------------- | ---
| Branch?       | 7.0
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| Tickets       | -
| License       | MIT
| Doc PR        | https://github.com/symfony/symfony-docs/pull/18781

Follow-up of https://github.com/symfony/symfony/pull/51425

### CODE REVIEW

File: src/Symfony/Bridge/Doctrine/composer.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
         "symfony/stopwatch": "^6.4|^7.0",
         "symfony/translation": "^6.4|^7.0",
         "symfony/uid": "^6.4|^7.0",
-        "symfony/validator": "^6.4|^7.0",
+        "symfony/validator": "^7.0",
```
Comment (nicolas-grekas) [inline src/Symfony/Bridge/Doctrine/composer.json]: until now, we managed to not do this anywhere
are these changes to fix existing failing tests or to accommodate changes made in this PR?
this PR should only contain what is required for it

Comment (alexandre-daubois) [reply src/Symfony/Bridge/Doctrine/composer.json]: You're right, all the classes needed in these components are already present in `symfony/validator ^6.4`. I've reverted these changes.

File: UPGRADE-7.0.md [hunk 2]
```diff
@@ -475,6 +477,9 @@ Validator
  * Remove the annotation reader parameter from the constructor signature of `AnnotationLoader`
```
Comment (derrabus) [inline UPGRADE-7.0.md]: This line should be removed. The whole class is gone now.

File: src/Symfony/Component/Serializer/Tests/Normalizer/AbstractObjectNormalizerTest.php [hunk 3]
```diff
@@ -866,9 +866,7 @@ public function __construct(
 
 class SnakeCaseNestedDummy
 {
-    /**
-     * @SerializedPath("[one][two_three]")
-     */
+    #[SerializedPath('[one][two_three]')]
```
Comment (derrabus) [inline src/Symfony/Component/Serializer/Tests/Normalizer/AbstractObjectNormalizerTest.php]: Can you revert the changes you've made to this file and submit them for 6.4, please?

**Review by derrabus — CHANGES_REQUESTED** [review]

File: UPGRADE-7.0.md (hunk 2 above)
Comment (alexandre-daubois) [reply UPGRADE-7.0.md]: Removed

File: src/Symfony/Component/Serializer/Tests/Normalizer/AbstractObjectNormalizerTest.php (hunk 3 above)
Comment (alexandre-daubois) [reply src/Symfony/Component/Serializer/Tests/Normalizer/AbstractObjectNormalizerTest.php]: Here you go https://github.com/symfony/symfony/pull/51465 :slightly_smiling_face:

**Comment by nicolas-grekas** [comment]
Thank you @alexandre-daubois.
